source/clear-linux/tutorials/kubernetes.rst

```diff
@@ -0,0 +1,299 @@
+  .. _kubernetes:
+
+Run Kubernetes\* on Clear Linux
+###############################
+
+This tutorial describes how to install, configure, and run the 
+`Kubernetes system`_ on |CL-ATTR| using either the Kubernetes
+runtime interface *CRI-O* and *kata-runtime* or *Docker\** and *runc*.
+
+Kubernetes\* is an open source system for automating deployment, scaling, and
+management of containerized applications. It groups containers that make up an
+application into logical units for easy management and discovery.
+
+Runc and kata-runtime are compatible with :abbr:`OCI (Open Container
+Initiative)` guidelines and work seamlessly with Kubernetes. `Kata
+Containers`_\* provide strong isolation for untrusted workloads or
+multi-tenant scenarios.  Runc and Kata Containers can be allocated on a
+per-container basis so you can mix and match both on the same host to suit
```

Comment:
I would say on a per-pod basis, since this is the smallest schedulable unit in Kubernetes, and also the granularity at which you can select a runtime for (ie, runc or kata)